src/SWP/SuperdeskBridgeBundle/Resources/config/services.yml

```diff
@@ -0,0 +1,9 @@
+services:
+
+    swp_sd_bridge.bridge:
+        class: SWP\SuperdeskBridgeBundle\Bridge\Bridge
+        arguments: [@swp_sd_bridge.client, %superdeskbridge%]
+
+    swp_sd_bridge.client:
+        class: SWP\SuperdeskBridgeBundle\Client\Client
+        arguments: [%superdeskbridge%]
```

Comment:
we should provide special SystemPreferences Service witch will can read from storages (database, custom configuration files) and pass parameters to client from there (with http://symfony.com/doc/current/book/service_container.html#using-the-expression-language) 

example: 

```
arguments:    ["@=service('swp_system_preferences').getParameters('swp_sd_bridge.client')"]